Ghidra/Framework/Utility/src/main/java/ghidra/util/config/ReadConfigProperties.java

```diff
@@ -0,0 +1,34 @@
+/* ###
+ * IP: GHIDRA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ghidra.util.config;
+
+import ghidra.framework.ColorProperties;
+
+import java.awt.*;
+
+import static utility.application.ApplicationLayout.getConfigurationProperties;
+
+
+public class ReadConfigProperties {
```

Comment:
Why do we need this helper class (which has a REALLY BAD NAME) instead of just call `ApplicationLayout.getColorProperties().readColor(key);`?